scripts/core/itemList/LazyLoader.tsx

```diff
@@ -63,7 +63,7 @@ export class LazyLoader<T> extends React.Component<IProps<T>, IState<T>> {
     private reloadAllItems() {
         const MAX_PAGE_SIZE = 200; // back-end limit
 
-        const loadedItemsCount = this.state.items.size;
+        const loadedItemsCount = this.state.items.size > 0 ? this.state.items.size : 1;
```

Comment:
can you pls add some comment to this? not sure why you do it

Reply:
Done.